packages/alfa-rules/test/sia-r23/rule.spec.tsx

```diff
@@ -200,6 +200,35 @@ test(`evaluate() cannot tell if questions are left unanswered`, async (t) => {
   );
 });
 
+test(`evaluate() doesn't ask if audio is playing when autoplay attribute is present`, async (t) => {
+  const target = <audio src="foo.mp3" autoplay />;
+
+  const document = h.document([target]);
+
+  t.deepEqual(
+    await evaluate(R23, { document }, oracle({ "is-audio-streaming": false })),
+    [cantTell(R23, target, undefined, Outcome.Mode.SemiAuto)],
+  );
+});
+
+test(`evaluate() doesn't ask about play button if controls attribute is present`, async (t) => {
+  const target = <audio src="foo.mp3" controls />;
+
+  const document = h.document([target]);
+
+  t.deepEqual(
+    await evaluate(
+      R23,
+      { document },
+      oracle({
+        "is-audio-streaming": false,
+        "is-playing": false,
+      }),
+    ),
+    [cantTell(R23, target, undefined, Outcome.Mode.SemiAuto)],
+  );
```

Comment:
**Issue:** These should be examples where the outcome is **not** `cantTell` to emphasize that we can get a definitive outcome without answering that question.
(it would be Inapplicable without he change, but I feel it would be less confusing to reread if we see a Passed or Failed result in this case)

Reply:
Done